README.md

```diff
@@ -16,10 +16,10 @@ A minimal in-place interpreter for [WebAssembly](https://webassembly.org/) bytec
 
 ## Features
 
-- **In-place interpretation**: No intermediate parsing step. This allows for fast start-up times.
+- **In-place interpretation**: No intermediate representation, directly interprets WebAssembly bytecode. This allows for fast start-up times.
 - **`no_std` support**: The interpreter requires only Rust's `core` and `alloc` libraries allowing its use in various environments, such as bare-metal systems.
 - **Minimal dependencies**: The interpreter requires only two dependencies: `log`, `libm`.
-- **Compliance with specification**: The interpreter passes almost all test from the [official WebAssembly testsuite](https://github.com/WebAssembly/testsuite). Only SIMD instructions are missing.
+- **Compliance with specification**: The interpreter passes all tests, except for the proposals and SIMD, from the [official WebAssembly testsuite](https://github.com/WebAssembly/testsuite). See [`GlobalConfig` in `tests/specification/mod.rs`](tests/specification/mod.rs) for the default spec-test filter.
```

Comment:
```suggestion
- **Compliance with specification**: The interpreter passes all tests from the [official WebAssembly testsuite](https://github.com/WebAssembly/testsuite) with the exception of [WebAssembly proposals](https://github.com/WebAssembly/proposals). See [`GlobalConfig` in `tests/specification/mod.rs`](tests/specification/mod.rs) for the default spec-test filter.
```

Which proposals do you mean? SIMD is also a proposal https://github.com/WebAssembly/spec/blob/1041527d508218acc40f0278d4abc3be9ba5e3bd/proposals/simd/SIMD.md?plain=1#L7

Reply:
Good catch, reformulated it further